components/selection/selection-select-all.js

```diff
@@ -51,6 +51,7 @@ class SelectAll extends LocalizeCoreElement(SelectionObserverMixin(LitElement))
 			composed: true,
```

Comment:
Still not sure if we want this event or not, now that the mixin doesn't listen for it.

Reply:
I always lean towards not exposing things until we absolutely need them.

Reply:
Removed the event.  Will be easy to add back if it turns out a consumer does need it